opensearchapi/api_search-params_test.go

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// The OpenSearch Contributors require contributions made to
+// this file be licensed under the Apache-2.0 license or a
+// compatible open source license.
+//
+// Modifications Copyright OpenSearch Contributors. See
+// GitHub history for details.
+
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This is new code, remove the Elasticsearch license grant part, just leave the SPDX one.